code/modules/reagents/Chemistry-Reagents/Chemistry-Reagents-Drinks.dm

```diff
@@ -692,29 +692,29 @@
 		S.handle_sugar(M, src, 0.66)	//Maple syrup is about 2/3 sugar in real life
 
 /datum/reagent/drink/space_cola
-	name = "Space Cola"
+	name = "Coca-Cola"
 	description = "A refreshing beverage."
 	taste_description = "cola"
 	reagent_state = LIQUID
 	color = "#100800"
 	adj_drowsy = -3
 	adj_temp = -5
 
-	glass_name = "Space Cola"
-	glass_desc = "A glass of refreshing Space Cola"
+	glass_name = "Coca-Cola"
+	glass_desc = "A glass of refreshing normal cola."
```

Comment:
Bruh.

Reply:
Copyright infringement.

Reply:
Remove the word "normal" from cola.

Reply:
> Remove the word "normal" from cola.

There are different flavoured colas, so I thought this would be the default point.
glass_desc = "A glass of refreshing cola."